lectia-4.js

```diff
@@ -1,2 +1,53 @@
 // https://miorita.scoalaweb.ro/lessons/4-function
 // Scrie un program care să o ajute pe Miorița să ridice baltagul și să îl aducă Vitoriei
+
+
+//versiunea 1 de rezolvare
+
+
+function turnAround(){     
+    if(canMove()){
+        turnRight();
+        turnRight();
+    }
+    if(!canMove()){
+        turnRight();
+        move();
+        turnRight();
+    }
+}
+
+function walk(parametru){
+    for(let i=0;i<parametru;i++){
+        move();
+    }
+}
+function moveAroundFence(paramFence) {
+    for(let i=0;i<paramFence;i++){
```

Comment:
Anticipezi că va fi nevoie să faci această secvență de instrucțiuni de mai multe ori succesiv?

Dacă nu, aș zice că nu ai nevoie de `for`